schema/src/lib.rs

```diff
@@ -87,14 +84,24 @@ pub struct ArrayMeta {
     pub ty: String,
     /// Length
     pub len: u64,
+    /// Order elements
+    pub sorted: bool,
+}
+
+/// Array metadata
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize)]
+pub struct VecMeta {
+    /// Type
+    pub ty: String,
+    /// Order elements
+    pub sorted: bool,
```

Comment:
```suggestion
    pub sorted: Option<SortFn>,
```

this would be preferable but I don't know how to support such feature

Reply:
In order of preference 

1) Data used to construct a comparator. It should be the variant of the structure and the order of precedence of fields. We rarely if ever should need anything more complicated. 
2) serialise a lambda into WASM. Powerful, but overkill IMO. 
3) embedded Turing complete scripting language. Lua is the most common example. Could also, technically be combined with 2) if we consider WAT.
4) hard code for each sorted structure. More work for SDK maintainers.

Reply:
opened an issue:  #2116